src/main/resources/assets/api-docs.yaml

```diff
@@ -2079,12 +2079,12 @@ paths:
       responses:
         '200':
           description: Returns eRA Commons Account information. Map<String, String> format
-          schema:
-            $ref: '#/definitions/EraProperties'
+#          schema:
+#            $ref: '#/definitions/EraProperties'
```

Comment:
Note: need a story to fix this - `EraProperties ` is missing and it breaks swagger parsing.